### ISSUE TITLE
Just a quick question about the saf provider

### ISSUE THREAD

**Issue by tvdev7805** [wa2c/cifs-documents-provider#32]
is it possible for a client program to connect/communicate to cifs documents provider without using the system documentsui picker? because there is no picker in most of the android tv... thanks.

**Comment by tvdev7805** [comment]
Thanks, is it possible to write a custom picker? I noticed that the picker is called FILES and it handles action open tree intent action.

**Comment by RebelliousX** [comment]
I don't think so, the services are missing in Android TV, it has to be baked and built with the firmware. The only Android TV that has Files app is the Shield TV as far as I know.
